dfvfs/lib/apfs_helper.py

```diff
@@ -30,3 +30,21 @@ def APFSContainerPathSpecGetVolumeIndex(path_spec):
     volume_index = None
 
   return volume_index
+
+
+def APFSContainerOpenVolume(apfs_volume, path_spec, key_chain):
```

Comment:
This doesn't actually open the volume, unlike the similar methods for bde and fvde. Please rename the method (APFSContainerUnlockVolume?) and update the docstring.

Reply:
ack